quiz/src/main/java/com/epam/spring/homework3/quiz/controller/UserController.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.spring.homework3.quiz.controller;
+
+import com.epam.spring.homework3.quiz.controller.assembler.UserAssembler;
+import com.epam.spring.homework3.quiz.controller.assembler.model.UserModel;
+import com.epam.spring.homework3.quiz.controller.dto.UserDto;
+import com.epam.spring.homework3.quiz.controller.dto.group.OnCreate;
+import com.epam.spring.homework3.quiz.controller.dto.group.OnUpdate;
+import com.epam.spring.homework3.quiz.controller.mapper.UserMapper;
+import com.epam.spring.homework3.quiz.service.UserService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequestMapping("/user")
+@RequiredArgsConstructor
+@Api(tags = "UserController description for SWAGGER documentation")
+@ApiResponses({
+        @ApiResponse(code = 404, message = "Not found"),
+        @ApiResponse(code = 500, message = "Internal Server Error")
+})
+public class UserController {
+    private final UserService userService;
+    private final UserMapper userMapper;
+    private final UserAssembler userAssembler;
+
+    @ApiOperation("Get user by email")
+    @GetMapping(value = "/get/{email}")
```

Comment:
some remarks on rest URLs.

Basically that's not really good practice to include words "get", "delete", "update" directly into the url. Since we can use HTTP methods for this (HTTP PUT, HTTP DELETE, HTTP GET)
![image](https://user-images.githubusercontent.com/24360703/188267954-09f218cd-e12e-4d4d-83d5-e1313a2998b4.png)

you can read about this here:
https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api

So, update the URLs:
- getUserByEmail -> GET /user/{email}
- updateUserByEmail -> PUT/user/{email}
- deleteUserByEmail -> DELETE /user/{email}